cc-visionary/StickerMe
# Admin Sticker Editing: View, Functionality and Error Handling

## Tasks Done
- [x] Add Image
- You should do this through clicking the icon, then selecting the image ( the image's filename will be treated as the image's imageID)
- [x] Delete Image
- [x] Display the Added Images along with its corresponding imageID
- [x] Separate Display Images per Type (through the select tag)

## To Test:
1. Follow the instructions for setting up and running the project
2. Go to `http://localhost:8080/login`
3. Enter the `moderator` as the username and `password` as the password.
4. Then you'll automatically be redirected to `http://localhost:8080/admin` where you'll be able to see the admin feature editing functionality.

## Expected Output:
![image](https://user-images.githubusercontent.com/22016684/127771710-817a4a30-97f7-4521-b761-a62968642acb.png)
